css/DayPicker.scss

```diff
@@ -29,6 +29,10 @@
   color: $react-dates-color-placeholder-text;
   cursor: pointer;
   line-height: 0.78;
+  -moz-user-select: none;
+  -webkit-user-select: none;
+  -ms-user-select: none;
```

Comment:
This seems like the sort of thing that postcss should do (autoprefixing) instead of cluttering the raw source with it.